<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Surface_Modelisation_TabWidget</class>
 <widget class="QWidget" name="Surface_Modelisation_TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="13" column="1" colspan="2">
    <widget class="QPushButton" name="button_accept">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Operations :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="combo_vertexSelector">
     <item>
      <property name="text">
       <string>- select selector -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="button_applyVertexOperation">
     <property name="text">
      <string>Apply vertex operation</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="combo_edgeSelector">
     <item>
      <property name="text">
       <string>- select selector -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QComboBox" name="combo_generalOperation">
     <item>
      <property name="text">
       <string>- select operation -</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Create empty map</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Create new face</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Add cube</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Merge volumes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Split surface</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Extrude region</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Vertex selector :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Position :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Face selector :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="combo_positionAttribute">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>- select attribute -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Clicks :</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QComboBox" name="combo_edgeOperation">
     <item>
      <property name="text">
       <string>- select operation -</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cut edge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uncut edge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Collapse edge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flip edge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flip back edge</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="button_applyGeneralOperation">
     <property name="text">
      <string>Apply general operation</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QComboBox" name="combo_vertexOperation">
     <item>
      <property name="text">
       <string>- select operation -</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Split vertex</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Delete vertex</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="button_applyFaceOperation">
     <property name="text">
      <string>Apply face operation</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QPushButton" name="button_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Edge selector :</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QPushButton" name="button_start">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QComboBox" name="combo_faceOperation">
     <item>
      <property name="text">
       <string>- select operation -</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Split face</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Merge faces</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Delete face</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sew faces</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unsew faces</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Extrude face</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Extrude face following a path</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="button_applyEdgeOperation">
     <property name="text">
      <string>Apply edge operation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QComboBox" name="combo_faceSelector">
     <item>
      <property name="text">
       <string>- select selector -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
